site/snippets/templates/release-40/migration.php

```diff
@@ -18,7 +18,7 @@
 			<div class="prose">
 				<?= kirbytext(
 					'<info>' .
-								'If you are updating from Kirby 3.8 or older, please first perform each major update step by step. ' .
+								'If you are updating from Kirby 3.9 or older, please first perform each major update step by step. ' .
```

Comment:
If a site is already on 3.9, there is no need to update it step by step using the previous migration guides.